### DISCUSSION TITLE
그룹#3 회고

### DISCUSSION THREAD

**Discussion by godrm** [codesquad-members-2022/swift-photoframe#139] [Show and tell]
여기에 각자 이번주 회고를 기록해주세요.
- 학습하면서 잘했던 점, 좋았던 점은 무엇인가요? (다음주에도 계속 유지해보세요)
- 학습하면서 스스로 아쉬웠던 점, 부족했던 점은 무엇인가요 - 그래서 어떻게 개선할 수 있을까요? (다음주에는 개선해봐요)
- 새롭게 알게 되거나 시도하지 못했지만, 다음주에 도전해볼건 무엇인가요? (다음주에 시도해봐요)
- 학습하는 데 조금이라도 도움이 된 사람이 있다면 칭찬해주세요!

**Comment by TaeKyeongKim** [comment]
### [학습하면서 잘했던 점, 좋았던 점]
- 그룹원 분들이 그룹 리뷰와 참여해 주셔서 좋았고 질문도 잘 받아주셔서 감사했습니다.
- 새로운 계념과 작업 환경 세팅 하는 법을 배워서 좋았습니다. 

### [학습하면서 스스로 아쉬웠던 점, 부족했던 점]
- 새로운 계념을 배움으로 인해 미션 진행속도가 더뎌서 혼자 쫓기는 마음으로 미션을 진행했던것같습니다. 계념도 자세히 못보게되고 배웠던 내용들이 머리속에서 겉돌기만 했던것 같아 다음주는 쫓기지 말고 하나하나 설명할수 있을정도의 학습을 마친후 다음 미션으로 넘어가겠습니다.

### [새롭게 알게 되거나 시도하지 못했지만, 다음주에 도전해볼것]
- `Notion` 이라는 블로그? 과 Github 의 `TIL` 을 처음 알게 되었습니다. 다음주에는 새롭게 배우게되는 계념들을 이같은 도구를 사용해 기록을 해볼까 합니다. 
-  git 의 사용도 더욱 다양한 방법으로 시도해볼것같습니다. 

### [도움 주신분들] 
- 그룹 3 모든 인원분들이 질문에 적극적으로 응대해 주셔서 너무 감사하고 특히 , @Jinsujin , @shingha1124 님 따로 시간 내주셔서 git 에 늪에 빠진 저를 도와주셔서 감사합니다.

**Reply by BumgeunSong** [reply]
댓글 원정 왔습니다

노션 입문 축하드립니다 🎊

**Reply by LeeJoobang** [reply]
깃을 함께 뿌시러 가시죠 :fire:

**Reply by Jinsujin** [reply]
git 은 계속 사용해 보면 익숙해 질거예요! 다음주에도 화이팅 입니닷

**Comment by sally4405** [comment]
## 학습하면서 잘했던 점, 좋았던 점
- 드디어 git이 에러를 일으키지 않고 있어요..! 이게 좋은건지 나쁜건지는 모르겠지만! 

## 학습하면서 스스로 아쉬웠던 점, 부족했던 점
- 시간 관리를 실패해서 낮과 밤이 엄청 바뀌고 제대로 미션을 수행하지 못한 것 같습니다ㅠㅠ
- 익숙하던 것 말고 새로운 방식으로 시도해보라는 피드백을 지속적으로 받고 있는데, 새로운 방법을 찾아보고 시도해보지 못한 것이 아쉬웠습니다😢

## 새롭게 알게 되거나 시도하지 못했지만, 다음주에 도전해볼 것
- 다음주부터는 덜 익숙한 것들 일거 같지만, 그래도 익숙한게 있다면 최대한 새로운 방식을 찾아보고 도전해보겠습니다..!
- pr 머지 되기 전에 지금은 에러가 안나는 익숙한 방법만 사용해서 브랜치를 관리하고 있는데, 다음주부터는 좀 더 실험적으로 브랜치를 관리 해보고 싶습니당

## 도움 주신 분들
- 저희 그룹원들 모두 소통에 적극적이셔서 아주 좋았습니다! 제 문제가 아니더라도 다른 분들 문제를 해결해주시는 과정을 보고도 많이 배웠습니다😄

**Reply by LeeJoobang** [reply]
왜 샐리의 깃은 말을 잘듣죠....?

**Reply by Jinsujin** [reply]
샐리...이번주에는 제가 상태가 안좋으면 샐리가 쌩쌩하고..그랬던거 같아 좀 더 얘기를 못나눠봐서 아쉬워요.
질문할때 세심하게 알려주셔서 너무나 감사했습니다! 
잠못드는 사람이 여기 또 있구나..동지애를 느꼈네요, 우리 몸 건강 챙기면서 해요!

**Comment by wnsxor1993** [comment]
## [학습하면서 잘했던 점, 좋았던 점]
- 모르는 부분을 물어보는 데에 있어서 약간의 부담감이 있었지만 이번에는 전보다는 좀 더 적극적인 느낌으로 종종 질문을 했다는 점이 뿌듯하게 느껴졌습니다.
- 최대한 현재 공부하는 내용과 연관되거나 파생되는 내용도 같이 공부해보려는 자세가 잡힐 듯하는 이 느낌이 좋은 징조 중 하나지 않을까 생각했습니다.

## [학습하면서 스스로 아쉬웠던 점, 부족했던 점]
- 여러가지 연관 및 파생 개념도 공부한다고 했지만 깊이감 없이 얕게 습득하고 지나가는 건 아닐까라는 생각이 자꾸 듭니다.
- 하나의 개념을 토대로 어디까지 범주를 넓히고 깊게 파고들어가야 될 지 전혀 감이 서질 않습니다. 그러다보니 점점 더 딥하거나 복잡한 개념까지 뻗어나가면 얼른 손을 떼버리는 것 같은 기분이 듭니다.

## [새롭게 알게 되거나 시도하지 못했지만, 다음주에 도전해볼것]
- git 사용과 관련해서 여러 방법들을 시도하는 걸 해보고 싶습니다
- 자꾸 commit을 잘게 쪼개는 걸 까먹는데 다음주는 까먹지 않고 실천할 수 있도록 하겠습니다

## [도움 주신분들]
항상 그룹 내에서 많은 정보들을 공유받을 수 있어서 감사한 마음입니다.
특히 UIKit이 처음이라 버벅이는 저한테 친절한 강의를 들려준 Mase한테도 감사하다는 인사 전합니다^^

**Reply by Jinsujin** [reply]
질문하는 것도 생각을 정리해나가면서 해야 하기때문에 어렵죠...! 
너무 깊게 학습하는 부분도 현재 공부하고 있는 주제에 벗어나지 않게 바운더리를 잘 잡아야 하는데, 이것 또한 어려운 문제인것 같습니다 ㅜㅜ 저도 너무 멀리 가버리곤 합니다...계속 학습하며 되돌아보는 시간을 가지면 어느정도 기준이 생기지 않을까 싶어요!

**Comment by LeeJoobang** [comment]
<h3> :pencil2: 학습하면서 잘했던 점, 좋았던 점</h3>

> <h3> 동료들의 정리 방법 등을 참조하여 나만의 정리 방법을 추가할 수 있었습니다.</h3>
> <h3>깃을 통해 정리를 시작하게 되어 학습 범위에 대해 보다 분명하게 알 수 있어 좋았습니다.</h3>

<h3> :pencil2: 학습하면서 아쉬웠던 점, 부족했던 점</h3>

> <h3>주어진 과제 및 추가 학습에 대한 부분까지 포함하여 완료하고 싶었으나 해당 범위까지 공부하지 못해서 아쉬웠습니다.</h3>

<h3> :pencil2: 새롭게 알게 되어 시도하지 못했지만, 다음 주에 도전해볼 건?</h3>

> <h3>깃 사용시 터미널을 사용할 수 있음을 알 수 있었으나 사용해보진 못했습니다. 해당 내용에 대해 도전하겠습니다.</h3>

<h3> :pencil2: 학습하는데 도움주신 분들</h3>

> <h3> 그룹 내에서 질의한 부분에 대해 모두 답변을 해주셨습니다. 먼저 Git 사용 플로우에 대해 '싱하'를 통해 배울 수 있었습니다. 
> <h3> 학습 정리 방법 등은 '로사'를 통해 배울 수 있었습니다. 감사합니다.</h3>

**Reply by Damagucci-Juice** [reply]
모니터 구매 축하드립니다. 총총

**Reply by BumgeunSong** [reply]
압도적인 연필 이모지 사이즈.. 데코 능력이 상당하시네요

**Reply by JasonLee0223** [reply]
모니터 구매를 하셨군요!! ㅎㅎ 저는 맥미니를 구입했습니다!! 주방 같이 힘내요!! 💪🏻💪🏻

**Reply by sungju-kim** [reply]
모니터 !!🎉🎉🎉🎉

**Reply by Jinsujin** [reply]
모니터 구입 축하드려요 🥳  이제 키보드 가시죠

**Comment by SangHwi-Back** [comment]
# 학습하면서 잘했던 점, 좋았던 점

* 저는 Git을 모른다는 걸 알았습니다.
* 정리를 해야지 해야지 하면서도 사실 내키지 않아 소홀히 했던 것이 사실입니다. 이번에는 대부분의 시간을 탐색, 정리로 보내보았는데 정말 얕게 알고 있었단 걸 알아서 좋았습니다.

# 학습하면서 스스로 아쉬웠던 점, 부족했던 점

* 저는 Git을 모른다는 걸 알았습니다.
* 정리만 너무 한 느낌입니다. 코딩테스트 연습도 하고 이전에 연습해 놓았던 것들도 잃지 않으려고 애써야 할텐데(적어도 iOS 개발 관련해서는) 너무 한쪽으로만 쏠리는 느낌이었습니다.
* 뭔가 안 풀리면 무지성으로 명령어를 입력해버리는 습관이 있다는 걸 다시한번 깨달았습
* 읽을 책도 많고 코딩테스트 문제풀이도 많이 해봐야 하는데 못하고 있습니다. CS10 때 열심히 해놓을걸 하는 생각도 듭니다.

# 새롭게 알게 되거나 시도하지 못했지만, 다음주에 도전해볼 것

* 학습 중간에 코딩테스트 문제풀이 시간을 따로 정할겁니다. 저녁에 코딩테스트 문제 풀이를 할까 했는데 공부하던 게 자꾸 아른거려서 해보지 못하고 하루를 넘겨버리고 있습니다.

# 학습하는 데 조금이라도 도움이 된 사람이 있다면 칭찬해주세요!

* 가끔 이상한 소리도 했는데 그래도 좋은 분위기였던 것 같아서 그룹 3 모든 분들께 감사드립니다.

**Reply by BumgeunSong** [reply]
가장 큰 배움: Git을 모른다는 걸 알았다!!

**Reply by Jinsujin** [reply]
꾸준히 운동도 병행하시고... 벡... 사람으로서 존경합니다 🙏🏼  자신을 잘 컨트롤 한다는 느낌이 들어 멋있어 보였어요! 저도 운동(산책이지만) 꾸준히 해서 사람이 되겠습니다

**Comment by OceanShape** [comment]
### 학습하면서 잘했던 점, 좋았던 점
- git에서 문제가 많았지만, 과제 자체는 설명이 잘 적혀 있어서 편하게 진행할 수 있었습니다.

### 학습하면서 스스로 아쉬웠던 점, 부족했던 점과 개선 방법
- 목요일 오전에 수업을 빠지게 되었는데, 강의 자료를 보니 궁금했던 내용이 많아 빠진 게 못내 아쉬웠다😭
- git을 사용하는 데에 어려움이 많았다.
- 공부한 내용에 대해 정리가 허술하다는 생각이 들었습니다. 다른 분들 레포지토리를 참고해서, 꼼꼼하게 정리하는 습관을 들여야겠습니다.

### 새롭게 알게 되거나 시도하지 못했지만, 다음주에 도전해볼 것
- 소스트리를 활용하여 git을 관리하고 싶었으나, 새 브랜치를 만드는 부분에서 충돌이 발생해 아직까지 못 써보고 있습니다. 다음주에는 소스트리로 버전 관리를 해보고 싶습니다.

### 학습하는 데 조금이라도 도움이 된 사람이 있다면 칭찬해주세요!
- 그룹원들이 먼저 고생해주셔서(?) 시간 낭비를 훨씬 덜 했습니다. 모두들 감사드려요😌

**Reply by BumgeunSong** [reply]
![](https://jjalbang.today/files/jjalbox/2022/01/20220130_61f6c53165281.jpg)

**Reply by Jinsujin** [reply]
다들 git 에 고생하시는 군요 ㅜㅜ 다음주에는 더 잘할 수 있도록 화이팅 입니닷

**Comment by Jinsujin** [comment]
### 학습하면서 잘했던 점, 좋았던 점은 무엇인가요?
- 단순히 사용하기에 급급했던 객체들에 대해 apple document 를 읽어보며 차근차근 알아갔습니다
- git 또한 항상 쓰던 기능에서 협업을 하며 만나는 문제들을 알아보고 해결 해 나간점이 좋았습니다

### 학습하면서 스스로 아쉬웠던 점, 부족했던 점은 무엇인가요 - 그래서 어떻게 개선할 수 있을까요?
- 이론 정리에 많은 시간이 들어서 실습을 많이 하지 못했습니다.
- 이론 정리시간, 실습시간을 하루에 얼마나 사용할지 분배하려고 합니다
- 다음주에는 하루 공부시간중 정리 시간 6: 실습 4 의 비율로 하려고 합니다

### 새롭게 알게 되거나 시도하지 못했지만, 다음주에 도전해볼건 무엇인가요? (다음주에 시도해봐요)
- 이번주에는 터미널에서 git merge시 발생하는 conflict 해결 방법을 알게 되었습니다.
- 다음주에는 터미널이 아닌 소스트리를 사용해서 git 사용을 시도해 보려고 합니다

### 학습하는 데 조금이라도 도움이 된 사람이 있다면 칭찬해주세요!
- 소스트리를 사용하는데 있어 싱하님께 많이 배웠습니다, 감사합니다!
- 샐리에게도 git 에 대해 많은 도움을 받았습니다! pr 받은 이후에 어떤 흐름으로 해야할지 개념정립에 도움이 되었습니다. 이번에 처음 같은 팀이 되었는데 컨디션 저조로 얘기를 많이 나눠보지 못해 아쉬워요,

**Reply by LeeJoobang** [reply]
이번주에 많이 배울 수 있었습니다 :-) 백신 회복 화이팅입니다!!ㅎㅎ

**Reply by TaeKyeongKim** [reply]
마지막날에 감사하다는 말을 제대로 못해서 아쉬웠어요 ㅠㅠ 빨리 회복하시고 다음주에도 파이팅 ! :)
